### PR TITLE
Blip shows only player with bus job

### PR DESCRIPTION
**Pull request by Tomboru**
So with my version the blips only get created for the player, when they have the bus job.
Also when they change the job, the blip gets removed.

I tested this Code, and its working without any problems and this code should fit for QB

### CODE REVIEW

File: client/main.lua [hunk 1]
```diff
@@ -260,20 +283,28 @@ RegisterNetEvent('qb-busjob:client:DoBusNpc', function()
...
-    SetBlipColour(BusBlip, 49)
-    BeginTextCommandSetBlipName("STRING")
-    AddTextComponentSubstringPlayerName(Lang:t('info.bus_depot'))
-    EndTextCommandSetBlipName(BusBlip)
+RegisterNetEvent('QBCore:Client:OnJobUpdate', function(JobInfo)
+    playerJob = JobInfo
+	DeleteBlip()
+    if playerJob.name == "bus" then
+		setupClient() --Creates the Blips
+    end
```
Comment (ItsANoBrainer) [inline client/main.lua:296]: Remove the job check here, as SetupClient already does it.

**Review by ItsANoBrainer — CHANGES_REQUESTED** [review]
Just one change then this is good to go.

**Comment by GhzGarage** [comment]
Has conflicts now, if you don't wanna fix let me know

**Comment by Tomboru** [comment]
> Has conflicts now, if you don't wanna fix let me know

Gonna do it now

**Comment by tom-osborne** [comment]
> > Has conflicts now, if you don't wanna fix let me know
> 
> Gonna do it now

Hi @Tomboru, this still has conflicts. Are you planning to fix them? 
I'd like to get this merged if possible.
Thanks

**Pull request closed without merging**